model_compression_toolkit/qunatizers_infrastructure/keras/config_serialization.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import copy
+
+from collections.abc import Callable
+
+from model_compression_toolkit import QuantizationConfig
+from model_compression_toolkit.core.common.quantization.node_quantization_config import NodeWeightsQuantizationConfig
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod, OpQuantizationConfig
+from enum import Enum
+
+IS_WEIGHTS = "is_weights"
+WEIGHTS_QUANTIZATION_METHOD = "weights_quantization_method"
+
+
+def transform_enum(v):
+    if isinstance(v, Enum):
+        return v.value
+    return v
+
+
+def config_serialization(quantization_config):
```

Comment:
Description, typehint (in all functions here, I think it's especially important here since it's not trivial to understand everything that happens)